source/operations/concepts/erasure-coding.rst

```diff
@@ -132,14 +116,11 @@ a MinIO deployment consisting of 1 node and 16 1TB drives:
 Storage Classes
 ~~~~~~~~~~~~~~~
 
-MinIO supports storage classes with Erasure Coding to allow applications to
-specify per-object :ref:`parity <minio-ec-parity>`. Each storage class specifies
-a ``EC:N`` parity setting to apply to objects created with that class. 
+MinIO supports storage classes with Erasure Coding to allow applications to specify per-object :ref:`parity <minio-ec-parity>`. 
+Each storage class specifies a ``EC:N`` parity setting to apply to objects created with that class. 
 
-MinIO storage classes are *distinct* from Amazon Web Services 
-:s3-docs:`storage classes <storage-class-intro.html>`. MinIO storage classes
-define *parity settings per object*, while AWS storage classes define *storage
-tiers per object*. 
+MinIO storage classes are *distinct* from Amazon Web Services :s3-docs:`storage classes <storage-class-intro.html>`. 
```

Comment:
Probably verbiage here that defines we are talking about Standard and RRS (which are AWS concepts) vs thier _additional_ definitions of storage class, ie with the end goal of avoiding confusion on S3 compatability